example/airtable/elden-ring-map/register.php

```diff
@@ -15,21 +15,35 @@
 function register_airtable_elden_ring_map_block() {
 	$block_name   = 'Elden Ring Location';
 	$access_token = \RemoteDataBlocks\Example\get_access_token( 'airtable_elden_ring' );
-	$base         = 'appqI3sJ9R2NcML8Y';
-	$tables       = [
-		'maps'      => 'tblS3OYo8tZOg04CP',
-		'locations' => 'tblc82R9msH4Yh6ZX',
-	];
 
 	if ( empty( $access_token ) ) {
 		$logger = LoggerManager::instance();
 		$logger->warning( sprintf( '%s is not defined, cannot register %s block', 'EXAMPLE_AIRTABLE_ELDEN_RING_ACCESS_TOKEN', $block_name ) );
 		return;
 	}
 
-	$elden_ring_datasource = new AirtableDatasource( $access_token, $base, $tables );
-	$list_locations_query  = new AirtableEldenRingListLocationsQuery( $elden_ring_datasource );
-	$list_maps_query       = new AirtableEldenRingListMapsQuery( $elden_ring_datasource );
+	$locations_config = [
+		'friendly_name'   => $block_name . ' (Locations)',
+		'uid'             => 'appqI3sJ9R2NcML8Y/tblc82R9msH4Yh6ZX',
```

Comment:
I'm confused about `uid` vs `uuid` vs `slug`. Is `uid` used?

Reply:
Ignore the payload atm. This is an experiment on top of the other work so the fields are quite malleable to say the least. I just wanted to get it up to discuss while keeping the code working.